packages/account/src/Components/currency-selector/radio-button.jsx

```diff
@@ -39,7 +39,16 @@ const USTPopover = ({ id }) => {
 };
 
 // Radio input
-const RadioButton = ({ field: { name, value, onChange, onBlur }, id, label, className, ...props }) => {
+const RadioButton = ({
+    field: { name, value, onChange, onBlur },
+    icon,
+    id,
+    label,
+    secondLineLabel,
```

Comment:
```suggestion
    second_line_label,
```
Since it is not a function.